gc/base/GlobalCollector.hpp

```diff
@@ -100,7 +100,13 @@ class MM_GlobalCollector : public MM_Collector {
  	*/
 	virtual void deleteSweepPoolState(MM_EnvironmentBase* env, void* sweepPoolState) = 0;
 
-	virtual void checkColorAndMark(MM_EnvironmentBase* env, omrobjectptr_t objectPt) { Assert_MM_unreachable(); }
+	virtual void checkColorAndMark(MM_EnvironmentBase* env, omrobjectptr_t objectPt) {};
+
+	/* Size in bytes to be reserved in an allocation cache for the collector. Required for specific use cases (SATB TLH Premark) */
+	virtual uintptr_t reservedForGCAllocCacheSize() { return 0; }
+
+	/* Used to notify the collector during cache flush if GC had part of cache reserved (reservedForGCAllocCacheSize). */
```

Comment:
the comment should be updated to something simpler like:

Used to notify the collector that current TLH has been flushed (Or about to be flushed?, The method name starts with 'pre' what one may interpret as pre-flush).

Reply:
but you need to describe the params

Reply:
>but you need to describe the params

Missed this comment, making change now